bindings/python/manifold3d.cpp

```diff
@@ -568,28 +530,24 @@ NB_MODULE(manifold3d, m) {
            "as empty. Likewise, empty meshes may still show NoError, for "
            "instance if they are small enough relative to their precision to "
            "be collapsed to nothing.")
-      .def_prop_ro(
+      .def(
```

Comment:
we should probably state that this is a breaking change, i.e. property vs method.